website/server/controllers/api-v3/user.js

```diff
@@ -1495,7 +1496,7 @@ api.userReleaseMounts = {
 api.userSell = {
   method: 'POST',
   middlewares: [authWithHeaders()],
-  url: '/user/sell/:type/:key',
+  url: '/user/sell/:type/:key/:amount',
```

Comment:
I didn't thought about that, so what would be the best practice for this? since the `common.ops.sell` can work without the amount parameter.

- copy the api.userSell with a different url
- or use amount as query-param?